website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -2893,6 +2893,183 @@ No response body.
+
+The endpoint can also be used to change the type of the data retention policy on the workspace. Making a request with the new type of policy will automatically create a data retention policy of the new type and replace the existing workspace policy. For more information on the types of data retention policies, refer to [Data Retention Policy Types](/terraform/cloud-docs/api-docs/data-retention-policies#types-of-data-retention-policies).
+
+| Parameter       | Description                                                                                                                                                         |
+| --------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `:workspace_id` | The workspace ID to update the data retention policy for. Obtain this from the [workspace settings](/terraform/cloud-docs/workspaces/settings) or the [Show Workspace](#show-workspace) endpoint. |
+
+| Status  | Response                  | Reason(s)                                                   |
+| ------- | ------------------------- | ----------------------------------------------------------- |
+| [204][] | No Content                | Successfully updated the workspace data retention policy.   |
+| [404][] | [JSON API error object][] | Workspace not found, or user unauthorized to perform action |
+
+### Request Body
+
+This POST endpoint requires a JSON object with the following properties as a request payload.
+
+| Key path          | Type   | Default | Description                    |
+| ----------------- | ------ | ------- | ------------------------------ |
+| `data.type`       | string |         | A [data retention policy type](/terraform/cloud-docs/api-docs/data-retention-policies#types-of-data-retention-policies) |
+| `data.attributes` | object |         | The attributes for the [chosen policy type](/terraform/cloud-docs/api-docs/data-retention-policies#types-of-data-retention-policies) |
+
+### Sample Payload
+
+```json
+{
+  "data": {
+    "type": "data-retention-policy-delete-olders",
+    "attributes": {
+      "deleteOlderThanNDays": 33
+    }
+  }
+}
+```
+
+### Sample Request
+
+```shell
+$ curl \
+  --header "Authorization: Bearer $TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request POST \
+  --data @payload.json \
+  https://app.terraform.io/api/v2/workspaces/ws-UYv6RYM8fVhzeGG5/relationships/data-retention-policy
+```
+
+### Sample Response
+
+```json
+{
+  "data": {
+    "id": "drp-Kh74zpKVGNWbpugQ",
+    "type": "data-retention-policy-delete-olders",
+    "attributes": {
+      "delete-older-than-n-days": 33
+    },
+    "relationships": {
+      "target": {
+        "data": {
+          "id": "workspace-2",
+          "type": "workspaces"
+        }
+      }
+    }
+  }
+}
+```
+
+## Remove data retention policy
```

Comment:
Non-Blocking Nitpick (Same as above!): Should we titlecase this for consistency between the other headers and this one?:
```suggestion
## Remove Data Retention Policy
```